nba-database/nbaProjectDBS.sql

```diff
@@ -55,601 +54,601 @@ PRIMARY KEY (UserID)
-INSERT INTO tbl_player(PlayerID,FirstName,LastName,Wins,Losses,PreviousWins,PreviousLosses,PlayerWinPercent,Points,Rebounds,Assists,Steals,Blocks,MissedFieldGoals,MissedFreeThrows,TurnOvers) VALUES (1629875,'Xavier','Moon',4,3,0,0,0.571,40,11,14,5,2,22,1,3);
-INSERT INTO tbl_player(PlayerID,FirstName,LastName,Wins,Losses,PreviousWins,PreviousLosses,PlayerWinPercent,Points,Rebounds,Assists,Steals,Blocks,MissedFieldGoals,MissedFreeThrows,TurnOvers) VALUES (1630270,'Xavier','Sneed',5,2,0,0,0.714,5,4,0,0,0,5,0,0);
-INSERT INTO tbl_player(PlayerID,FirstName,LastName,Wins,Losses,PreviousWins,PreviousLosses,PlayerWinPercent,Points,Rebounds,Assists,Steals,Blocks,MissedFieldGoals,MissedFreeThrows,TurnOvers) VALUES (1630214,'Xavier','Tillman',34,15,31,28,0.694,234,153,55,46,15,105,23,32);
-INSERT INTO tbl_player(PlayerID,FirstName,LastName,Wins,Losses,PreviousWins,PreviousLosses,PlayerWinPercent,Points,Rebounds,Assists,Steals,Blocks,MissedFieldGoals,MissedFreeThrows,TurnOvers) VALUES (1629139,'Yuta','Watanabe',20,16,24,26,0.556,156,85,19,8,16,77,16,19);
-INSERT INTO tbl_player(PlayerID,FirstName,LastName,Wins,Losses,PreviousWins,PreviousLosses,PlayerWinPercent,Points,Rebounds,Assists,Steals,Blocks,MissedFieldGoals,MissedFreeThrows,TurnOvers) VALUES (1630582,'Yves','Pons',8,1,0,0,0.889,5,6,0,0,0,3,2,0);
-INSERT INTO tbl_player(PlayerID,FirstName,LastName,Wins,Losses,PreviousWins,PreviousLosses,PlayerWinPercent,Points,Rebounds,Assists,Steals,Blocks,MissedFieldGoals,MissedFreeThrows,TurnOvers) VALUES (1628380,'Zach','Collins',12,11,0,0,0.522,159,117,49,9,18,64,9,35);
-INSERT INTO tbl_player(PlayerID,FirstName,LastName,Wins,Losses,PreviousWins,PreviousLosses,PlayerWinPercent,Points,Rebounds,Assists,Steals,Blocks,MissedFieldGoals,MissedFreeThrows,TurnOvers) VALUES (203897,'Zach','LaVine',37,28,26,32,0.569,1605,302,292,41,22,608,55,168);
-INSERT INTO tbl_player(PlayerID,FirstName,LastName,Wins,Losses,PreviousWins,PreviousLosses,PlayerWinPercent,Points,Rebounds,Assists,Steals,Blocks,MissedFieldGoals,MissedFreeThrows,TurnOvers) VALUES (1630192,'Zeke','Nnaji',24,16,30,12,0.6,265,145,15,16,13,82,24,22);
-INSERT INTO tbl_player(PlayerID,FirstName,LastName,Wins,Losses,PreviousWins,PreviousLosses,PlayerWinPercent,Points,Rebounds,Assists,Steals,Blocks,MissedFieldGoals,MissedFreeThrows,TurnOvers) VALUES (1630533,'Ziaire','Williams',40,18,0,0,0.69,443,120,60,31,12,208,10,38);
-INSERT INTO tbl_player(PlayerID,FirstName,LastName,Wins,Losses,PreviousWins,PreviousLosses,PlayerWinPercent,Points,Rebounds,Assists,Steals,Blocks,MissedFieldGoals,MissedFreeThrows,TurnOvers) VALUES (1629597,'Zylan','Cheatham',0,1,0,0,0,0,0,0,0,0,3,0,0);
+INSERT INTO tbl_Players(PlayerID,FirstName,LastName,Wins,Losses,PreviousWins,PreviousLosses,PlayerWinPercent,Points,Rebounds,Assists,Steals,Blocks,MissedFieldGoals,MissedFreeThrows,TurnOvers) VALUES (203932,'Aaron','Gordon',44,28,29,21,0.611,1062,413,181,43,41,383,56,126);
```

Comment:
Great good use of capitalisation and using s at the end.

Reply:
thank.